octodns_edgecenter/__init__.py

```diff
@@ -412,6 +416,17 @@ def populate(self, zone, target=False, lenient=False):
         )
         return exists
 
+    def _exclude_disabled_records(self, records):
+        _return_records = []
+        for record in records:
+            _result_resource_record = []
+            for resource_record in record["resource_records"]:
+                if resource_record.get("enabled", True):
+                    _result_resource_record.append(resource_record)
```

Comment:
What happens if all the rr's are `eanbled=False`? Looks like `record["resource_records"]` would be `[]`. Is that valid?